cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
> I need access to the external DNS server, and that is not routable in our environment without access to the host network

Can you explain this a little more? Using hostNetwork is bad practice and should be avoided if at all possible.